README.md

```diff
@@ -106,6 +106,25 @@ $ helm dt wrap examples/mariadb/
  🎉  Helm chart wrapped into "/Users/martinpe/workspace/distribution-tooling-for-helm/mariadb-13.0.0.wrap.tgz"
 ```
 
+If your chart and docker images include artifacts such as signatures or metadata, you can also include them in the wrap using the `--fetch-artifacts` flag:
```

Comment:
Added some clarification there because we are really not moving linked metadata today (i.e. using the OCI `subject` field), so we should list the conventions we are taking.